idaes/apps/grid_integration/pricetaker/design_and_operation_models.py

```diff
@@ -80,6 +145,20 @@ def my_design_model(m, p_min, p_max, cost):
             doc="Dictionary containing arguments needed for model_func",
         ),
     )
+    CONFIG.declare(
+        "fixed_design_data",
+        ConfigValue(
+            domain=dict,
+            doc="Dictionary containing parameters associated with unit/process design",
```

Comment:
doc string is exactly same for "variable_design_data". Please update so the user can distinguish the purpose of having these as two separate config options

Reply:
Ah- there is a minor difference--- this mentions "parameters" while the latter mentions "variables"